packages/salesforcedx-utils/src/i18n/advancedLocalization.ts

```diff
@@ -148,7 +148,7 @@ export class LocalizationService {
       try {
         const message = this.messageBundleManager.loadMessageBundle();
         this._nls = new Localization(message);
-      } catch (error) {
+      } catch {
```

Comment:
ex of prefer-optional-catch-binding (if you aren't using the error, you can omit it from the catch).  Helps you think about whether you needed anything from the error or not.